leave-application

AnastasiaBndr opened the pull request.

AnastasiaBndr committed:
leave-application

Inanim4te commented:
У тебя margin для секции не адаптирован под разные устройства

Inanim4te commented:
И задай, пожалуйста, своей сеции `id="leave-application"`, у нас по тз некоторые якорные ссылки должны вести в твою секцию

SergeyPoly commented on src/css/sections/leave-application-section.css:
```diff
@@ -1 +1,341 @@
+  --primary-transition: 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  --font-color:rgba(255, 255, 255, 0.7);
+  --title-color:rgba(255, 255, 255, 1)
+}
```
усі кольори і transition вже винесені у змінні у файлі variables, використовуйте їх, ці приберіть з коду

AnastasiaBndr committed:
leave-application-2

AnastasiaBndr committed:
Update leave-application-section.css

AnastasiaBndr committed:
Update leave-application-section.css

AnastasiaBndr committed:
Merge branch 'main' of https://github.com/SergeyPoly/goit-markup-final-project into leave-application

AnastasiaBndr committed:
leave-application fix

- added common styles for section-title, span, section-description
- added with-offset class to leave-application-section, section-title and section-description
- deleted all unnessesary top margins
